code/addons/interactions/package.json

```diff
@@ -99,7 +112,7 @@
     "entries": [
       "./src/index.ts",
       "./src/manager.tsx",
-      "./src/preset/preview.ts",
```

Comment:
This could also have remained as before, but it was inconsistent with the rest of the addons, and it's nice for them all to follow the same pattern.  I did the same for `addon-outline` as well.